render-configs.sh

```diff
@@ -8,3 +8,9 @@ docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN broadinstitute/dsde-toolbox:latest \
     jq .data | tee /tmp/jade-dev-account.json | \
     jq -r .private_key > /tmp/jade-dev-account.pem
 
+BUFFER_CLIENT_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/dev/dev/buffer/client-sa
+BUFFER_CLIENT_SERVICE_ACCOUNT_OUTPUT_PATH=/tmp/buffer-client-sa-account.json
+
+docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN broadinstitute/dsde-toolbox:latest \
```

Comment:
do we not assume `vault` is installed on developer's machines?

Reply:
Hmm I'm not sure why this script was using a docker image to begin with, but I don't think it hurts!